Library/ViewModels/PledgePaymentMethodsViewModelTests.swift

```diff
@@ -14,7 +14,12 @@ final class PledgePaymentMethodsViewModelTests: TestCase {
   private let notifyDelegateApplePayButtonTapped = TestObserver<Void, Never>()
   private let notifyDelegateCreditCardSelected = TestObserver<String, Never>()
   private let notifyDelegateLoadPaymentMethodsError = TestObserver<String, Never>()
-  private let reloadPaymentMethods = TestObserver<[GraphUserCreditCard.CreditCard], Never>()
+  private let notifyDelegatePledgeButtonTapped = TestObserver<Void, Never>()
+  private let pledgeButtonEnabled = TestObserver<Bool, Never>()
```

Comment:
Are these two outputs still supposed to be here? They might have been removed on master.